src/components/Section/SubstanceUse/Drugs/OrderedTreatment.jsx

```diff
@@ -66,11 +66,20 @@ export default class OrderedTreatment extends ValidationElement {
     let selected = cb.value
     let list = [...((this.props.OrderedBy || {}).values || [])]
 
-    if (list.includes(selected)) {
-      list.splice(list.indexOf(selected), 1)
-    } else {
-      list.push(selected)
-    }
+      if (list.includes(selected)) {
+        list.splice(list.indexOf(selected), 1)
+      } else {
+        if (selected !== "None" && list.includes("None")) {
```

Comment:
Based on what I'm seeing in this code, it seems like a list with 'None' in it will never have any other values. If that is the case, I think we can probably combine the `if/else` statement here and put
something like:
```
if ((selected !== "None" && list.includes("None")) || selected === 'None') {
  list = [selected]
}
```

Reply:
Thanks!